### ISSUE TITLE
Update tifffile plugin implementation

### ISSUE THREAD

**Issue by jni** [scikit-image/scikit-image#3734]
@cgohlke [suggested](https://github.com/scikit-image/scikit-image/pull/3675#issuecomment-462021998) that our tifffile plugin is overly complicated, and that we should instead simply wrap `tifffile.imread` as so:

```python
def imread(fname, dtype=None, **kwargs):
    """ ... """
    if 'img_num' in kwargs:
        kwargs['key'] = kwargs.pop('img_num')
    tifffile.imread(fname, **kwargs)
```

**Comment by sciunto** [comment]
@soupault Did you find the time to start something for this issue?

**Comment by sciunto** [comment]
I guess this issue can be closed as in #4235 tiffffile has been removed.
